src/main/java/github/weichware10/toolbox/zoommaps/ZoomImage.java

```diff
@@ -4,6 +4,8 @@
 import java.io.FileNotFoundException;
 import java.io.IOException;
 import java.net.MalformedURLException;
+import java.nio.file.Paths;
+
 import javafx.geometry.Point2D;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'javafx.geometry.Point2D'